<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_keep_original">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep original translation text:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonConfirm">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>371</width>
     <height>94</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Word Vector</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="word_vector_path"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Translation Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="translation_model_path"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Generation Model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="generation_model_path"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>97</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="keep_original_translation_text_true">
      <property name="text">
       <string>True</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="keep_original_translation_text_false">
      <property name="text">
       <string>False</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_load_model_if_needed">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Load model when needed:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>97</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="load_model_if_needed_true">
      <property name="text">
       <string>True</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="load_model_if_needed_false">
      <property name="text">
       <string>False</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
